src/components/about/css/about.css

```diff
@@ -0,0 +1,11 @@
+.mainrow{
+    display: -ms-flexbox;
+  display: flex;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+  width: 100%; 
+                    padding-top: 10vh;
```

Comment:
Consistent indenting makes code more readable. Please use an agreed upon code-indenting tool in your IDE.